lib/fastlane/plugin/cordova/actions/cordova_action.rb

```diff
@@ -192,9 +192,8 @@ def self.available_options
             key: :build_number,
             env_name: "CORDOVA_BUILD_NUMBER",
             description: "Build Number for iOS and Android Keystore alias",
-            default_value: 0,
+            optional: true,
             is_string: false,
```

Comment:
I saw that `is_string` is deprecated, but it seems like the other config items are still using it and I wanted to keep consistent. I can switch to using `type` instead, if you're cool with `build_number` being declared slightly differently from the other config items.

As for choosing a specific type, I don't want it to just be `String`, it should accept a `String` or a `Numeric` (well, really an `Integer`). Just accepting a string would be fine, but it would be a breaking change because previous versions did accept numbers for `build_number`. I left `is_string` false, because that allows it to accept both strings and numbers. I don't know what the equivalent would be if I used `type` instead (maybe just not specifying `type` would be enough).